lib/config/presets/index.ts

```diff
@@ -218,6 +219,12 @@ export async function getPreset(
   }
   const { presetSource, repo, presetPath, presetName, tag, params } =
     parsePreset(preset);
+
+  const cacheKey = preset;
```

Comment:
```suggestion
  const cacheKey = `preset:${preset}`;
```

Reduce chance of it "colliding" with other uses of the memCache